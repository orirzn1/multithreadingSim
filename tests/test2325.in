L12
S6
O14
N5

T0
I@0x00000000
ADDI $5, $4, 0x4
SUB $6, $6, $3
SUBI $7, $1, 0x3
STORE $1, $4, 0x2
SUB $3, $4, $3
ADDI $5, $7, 0x2
ADD $5, $6, $2
ADD $4, $7, $5
HALT $0

T1
I@0x00000000
SUBI $7, $1, 0x3
ADD $1, $3, $5
SUBI $7, $7, 0x3
ADDI $0, $2, 0x4
LOAD $6, $2, $4
SUB $2, $1, $7
ADDI $4, $5, 0x2
HALT $0

T2
I@0x00000000
ADDI $1, $7, 0x1
LOAD $4, $2, 0x0
SUBI $3, $0, 0x2
SUBI $6, $5, 0x4
LOAD $4, $5, 0x2
ADDI $5, $3, 0x1
HALT $0

T3
I@0x00000000
ADD $3, $5, $7
ADD $2, $2, $1
SUBI $0, $5, 0x4
SUB $1, $0, $7
ADDI $5, $0, 0x1
HALT $0

T4
I@0x00000000
ADDI $0, $2, 0x1
SUBI $0, $4, 0x2
ADDI $0, $5, 0x2
SUB $4, $4, $2
STORE $7, $3, $3
ADD $0, $5, $6
HALT $0

D@0x0
0xA
0x3
0x5
0x6
0x1
0x8
0x9
0x4
