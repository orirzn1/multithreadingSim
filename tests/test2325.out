
---- Blocked MT Simulation ----

Register file thread id 0:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0xFFFFFFFD	R5 = 0x0	R6 = 0x0	R7 = 0xFFFFFFFD
Register file thread id 1:
	R0 = 0x4	R1 = 0x0	R2 = 0x6	R3 = 0x0	R4 = 0x2	R5 = 0x0	R6 = 0x0	R7 = 0xFFFFFFFA
Register file thread id 2:
	R0 = 0x0	R1 = 0x1	R2 = 0x0	R3 = 0xFFFFFFFE	R4 = 0x0	R5 = 0xFFFFFFFF	R6 = 0xFFFFFFFC	R7 = 0x0
Register file thread id 3:
	R0 = 0xFFFFFFFC	R1 = 0xFFFFFFFC	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0xFFFFFFFD	R6 = 0x0	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Blocked MT CPI for this program 4.405405

-----Finegrained MT Simulation -----

Register file thread id 0:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0xFFFFFFFD	R5 = 0x0	R6 = 0x0	R7 = 0xFFFFFFFD
Register file thread id 1:
	R0 = 0x4	R1 = 0x0	R2 = 0x6	R3 = 0x0	R4 = 0x2	R5 = 0x0	R6 = 0x0	R7 = 0xFFFFFFFA
Register file thread id 2:
	R0 = 0x0	R1 = 0x1	R2 = 0x0	R3 = 0xFFFFFFFE	R4 = 0x0	R5 = 0xFFFFFFFF	R6 = 0xFFFFFFFC	R7 = 0x0
Register file thread id 3:
	R0 = 0xFFFFFFFC	R1 = 0xFFFFFFFC	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0xFFFFFFFD	R6 = 0x0	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Finegrained Multithreading CPI for this program 1.135135

